core/mavlink_parameters.cpp

```diff
@@ -264,7 +264,7 @@ void MAVLinkParameters::process_param_value(const mavlink_message_t &message)
         return;
     }
 
-    if (strncmp(work->param_name.c_str(), param_value.param_id, PARAM_ID_LEN) != 0) {
+    if (work->param_name.compare(param_value.param_id) != 0) {
```

Comment:
I'm not sure I get the difference. What was the issue with the previous one?

Reply:
Have you read the commit description?

Reply:
I didn't :sweat_smile:. I should get used to that, but I'm unfortunately not used to finding useful information in commit descriptions. Will work on it, thanks!

Reply:
Hehe, I try to write make them explanatory.